manifests/autotune/performance-profiles/resource_optimization_local_monitoring_norecordingrules.json

```diff
@@ -0,0 +1,395 @@
+{
+  "apiVersion": "recommender.com/v1",
+  "kind": "KruizePerformanceProfile",
+  "metadata": {
+    "name": "resource-optimization-local-monitoring"
```

Comment:
Do we want the name to be same as the other JSON?

Reply:
As we are using the same performance profile name while creating the experiments for openshift and minikube in kruize-demos, I have kept the name same.